.github/renovate.json5

```diff
@@ -0,0 +1,45 @@
+{
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  extends: [
+    'config:recommended',
+    'default:pinDigestsDisabled',
+    'security:openssf-scorecard'
+  ],
+  baseBranchPatterns: [
+    'main',
+  ],
+  configMigration: true,
+  prHourlyLimit: 1,
+  prConcurrentLimit: 3,
+  schedule: [ // utc timezone
+    'after 4pm on monday',
+  ],
```

Comment:
Schedule daily initially? https://docs.renovatebot.com/presets-schedule/#scheduledaily

If it becomes too noisy, we can reduce the schedule frequency later.

No limits on concurrency - this is not the main EarthBuild/earthbuild repo. workflows fast enough.